.github/build-and-test

```diff
@@ -42,7 +43,5 @@ do
         let ++FAILED
         FAILURES+=( "$dir" )
     fi
-    bazel shutdown
```

Comment:
The point here is that a Bazel server is started for every sub-directory. So if you postpone that until the end you have 8 Bazel servers running.